jsonb/src/main/java/io/avaje/jsonb/spi/JsonbExtension.java

```diff
@@ -0,0 +1,6 @@
+package io.avaje.jsonb.spi;
+
+import io.avaje.spi.Service;
+
+@Service
```

Comment:
I believe that `@Service` is only here for documentation purposes. I think it's almost clearer if we remove it as ALL Jsonb SPI services need to be detected and registered via avaje-jsonb-generator (and not avaje-spi-service).

I'll remove the `@Service` from here.